sources/engine/Stride.Engine/Engine/Startup/GameBuilder.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Reflection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+
+namespace Stride.Engine.Startup;
+
+public interface IGameBuilder
+{
+    public ConfigurationManager Configuration { get; }
+    public IServiceCollection Services { get; }
+    public ILoggingBuilder Logging { get; }
```

Comment:
ILoggingBuilder standardizes a way of configuring where to log to.
Many NuGet packages exist that work against this standard, from simple things like
files, console, Windows Event Log, to professional products like
Splunk or Sentry.

Even Slack and Telegram apparently... Maybe those are for something else.
Not gonna look into that, I doubt these will matter :D

In objects created by IServiceProvider, an ILogger<> can be received like this:
https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-7.0#create-logs

However, it will take time to move services over to IServiceProvider.
If we hook up an ILogger<> to Stride's existing logging mechanism,
people could already use all those Nuget packages mentioned above.